addons/website_slides_forum/models/slide_channel.py

```diff
@@ -25,12 +25,11 @@ def action_redirect_to_forum(self):
 
         return action
 
-    @api.model
-    def create(self, vals):
-        channel = super(Channel, self.with_context(mail_create_nosubscribe=True)).create(vals)
-        if channel.forum_id:
-            channel.forum_id.privacy = False
-        return channel
+    @api.model_create_multi
+    def create(self, vals_list):
+        channels = super(Channel, self.with_context(mail_create_nosubscribe=True)).create(vals_list)
+        channels.forum_id.privacy = False
```

Comment:
missing check that `forum_id` is set

Reply:
Calling write on an empty recordset is supported (and has no impact)